HTML Exercise/index.html

```diff
@@ -25,6 +25,18 @@
          One of the links should lead to an external website (use "#" for the rest) -->
     <nav>
       <!-- Code here -->
+      <div class="navbar-left">
+        <a href="#">Categories</a>
+        <a href="#">Deals</a>
+        <a href="#">Help</a>
+      </div>
+      <div class="navbar-right">
+        <a href="#">Sign Up</a>
+        <a href="#">Login</a>
+        <a href="#">Wishlist</a>
+        <a href="https://www.leniolabs.com/culture/">About us</a>
```

Comment:
Can you make it so that it opens the link on a new page?

Hint: Use the target attribute 👀

Reply:
done :)

Reply:
Buenisimo!